tests/init/test_init.py

```diff
@@ -714,10 +714,7 @@ def test_init_template_with_python_task_fails_on_missing_python(
     verify(result.output, scrubber=make_output_scrubber())
 
 
-def test_init_template_with_python_task_works(
-    which_mock: WhichMock, dummy_algokit_template_with_python_task: dict[str, Path]
-) -> None:
-    which_mock.remove("python")
```

Comment:
@negar-abbasi i haven't seen any effects of removing this i assume this was just a typo?

Reply:
@negar-abbasi i saw you restored the change back - can you explain why we need which_mock.remove('python') bit in this test?